### DISCUSSION TITLE
利用AI偵測呼吸道病毒

### DISCUSSION THREAD

**Discussion by VictorLin2002** [1101-GS3073-C/Python-Community-GS3073-C#666] [General]
在今年美國CES展中，科技公司Opteev Technologies展出運用AI技術開發的一款口袋大病毒檢測器ViraWarn，只要對儀器吹氣兩次，便可在60秒內檢測出COVID-19、RSV和流感，且可重複使用，這個儀器目前已經在等待美國FDA審核。其利用抓取病毒表面的電荷數據，並以AI分析，濾除錯誤讀數，即可知呼吸道是否被病毒感染。該公司希望藉此提供更方便及更舒適的感染檢測方法，進而減少使用抗原檢測相關的侵入式檢測所造成的不適。
https://opteev.com/science/
個人觀點:雖然聽起來很有趣，但事實上感覺並不實用，因為似乎無法辨識是不是COVID-19，僅能辨別是否有病毒。不過AI技術在生理健康相關檢測的產品還不成熟，這部分也是值得發展的部分。如果搭配AI，自己在檢測身體健康時，就可以有更客觀的參考，讓我們決定是否要就醫。

**Comment by bww0401** [comment]
我認為如果能特別偵測單一病毒會更好，畢竟如果只知道自己生病，不僅會增加人民的恐慌，還會增加醫療負擔。目前醫療AI已經有很多產品，包括偵測血糖、超音波判讀等等，希望未來能有更多醫療上的應用!

**Comment by 0212Jerry** [comment]
我覺得這項產品之所以感覺不怎麼準的原因可能是該公司基於隱私問題無法取得足夠的樣本進行分析，並且讓開發的Ai有足夠的機會訓練與學習。但要取得個人的樣本本來就有困難存在，個人認為這項工作與衛生單位結合會有更好的效果。此外，我個人建議，加入其他常見疾病一併進行分析會更好，若能針對檢測出來的結果給使用者相關的建議或搜尋使用者所在位置的附近醫療機構，如此一來不僅可以減輕現行醫療量能的負擔，也可以讓疾病更即時的受到處理。

**Comment by liangyinglly** [comment]
若醫生搭配AI，醫療用品也搭配AI，那麼對於病患的診斷會更精準，人們也可以及早發現及早治療，我認為醫療用品搭配AI這方面的技術很值得我們深入研究，所以這間公司應該還是很有潛力的！

**Comment by Vincent20030314** [comment]
這項發明很有發展潛力，感謝分享。

**Comment by HsuChen88** [comment]
這篇被給眼睛真令人遺憾，而且我覺得這款檢測器是一個跨世代的發明，很期待未來能拓展到所有病毒檢測上。
